### ISSUE TITLE
✅ 🐞 Bug Report: Inline css not correctly including responsive styles when theme templates in use

### ISSUE THREAD

**Issue by eastriverstudio** [elementor/elementor#18604]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When using theme builder templates (e.g a single page template), pages which use the template aren't getting the responsive styles printed to the page correctly. I seem to get both an inline style sheet and a page specific external file loaded, but the inline one doesn't contain the correct rules for responsive styles set in widgets, columns and sections and they override the correct values in the external file.  

Seems like an issue related to the css loading experiments but turning these off doesn't do anything and the inline styles are still printed to the page.

### Steps to reproduce

1. Create a template for all pages in the theme builder 
2. Add a post content widget to the page to display content
3. Apply template to all pages using the display rules
4. Make sure css loading experiments is on in settings > Experiments
5. Create a new page in Elementor with some widgets/sections/columns with mobile and tablet specific styles 
6. Check if all looks OK on the frontend.
7. Go back and edit the page in any way
8. Check again and see if responsive styles look correct for mobile/tablet

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v5.5.61-MariaDB-38.13
	PHP Version: 7.4.29
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 50M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.9.3
	WP Multisite: No
	Max Upload Size: 50 MB
	Memory limit: 256M
	Max Memory limit: 256M
	Permalink Structure: /%category%/%postname%/
	Language: en-GB
	Timezone: Europe/London
	Debug Mode: Inactive

== Theme ==
	Name: GeneratePress
	Version: 3.1.3
	Author: Tom Usborne
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:99.0) Gecko/20100101 Firefox/99.0

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 5.12.2
		Author: Delicious Brains

	Advanced Editor Tools (previously TinyMCE Advanced)
		Version: 5.6.0
		Author: Automattic

	All-in-One WP Migration
		Version: 7.59
		Author: ServMask

	Code Snippets
		Version: 2.14.3
		Author: Code Snippets Pro

	Cookiebot | GDPR/CCPA Compliant Cookie Consent and Control
		Version: 4.0.3
		Author: Cybot A/S

	Custom Post Type UI
		Version: 1.11.2
		Author: WebDevStudios

	DynamicConditions
		Version: 1.5.2
		Author: RTO GmbH

	e-addons DISPLAY
		Version: 2.0.4
		Author: Nerds Farm

	e-addons for Elementor
		Version: 3.0.9
		Author: Nerds Farm

	Elementor
		Version: 3.6.5
		Author: Elementor.com

	Elementor Pro
		Version: 3.6.5
		Author: Elementor.com

	FluentSMTP
		Version: 2.1.1
		Author: FluentSMTP & WPManageNinja Team

	GP Premium
		Version: 2.1.2
		Author: Tom Usborne

	Make Column Clickable Elementor
		Version: 1.4.0
		Author: Fernando Acosta

	Nested Pages
		Version: 3.1.20
		Author: Kyle Phillips

	Post Types Order
		Version: 1.9.9
		Author: Nsp Code

	ProfilePress
		Version: 3.2.11
		Author: ProfilePress Team

	Redirection
		Version: 5.2.3
		Author: John Godley

	Reinstall Plugins
		Version: 0.2
		Author: Nashwan Doaqan

	Safe SVG
		Version: 2.0.1
		Author: 10up

	Unlimited Elements for Elementor (Premium)
		Version: 1.5.12
		Author: Unlimited Elements

	Wordfence Security
		Version: 7.5.9
		Author: Wordfence

	Yoast SEO
		Version: 18.8
		Author: Team Yoast


== Elements Usage ==
	
	page : 3
		button : 7
		call-to-action : 1
		heading : 16
		spacer : 10
		testimonial-carousel : 1
		text-editor : 8
		video : 2
	section : 8
		button : 17
		divider : 4
		heading : 30
		icon-list : 10
		image : 5
		social-icons : 1
		text-editor : 7
		video : 2
	wp-post : 15
		button : 5
		image : 8
		spacer : 3
		text-editor : 25
		video : 1
	wp-page : 100
		accordion : 13
		button : 69
		call-to-action : 2
		counter : 1
		divider : 26
		google_maps : 1
		heading : 160
		html : 1
		image : 21
		image-box : 4
		image-gallery : 1
		spacer : 65
		testimonial-carousel : 1
		text-editor : 170
		ucaddon_ue_listing_grid : 1
		video : 2
	single-page : 3
		button : 2
		heading : 4
		theme-post-content : 1


== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved asset loading: Inactive
	Improved CSS Loading: Inactive
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active by default
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Active
	Hide native WordPress widgets from search results: Active by default
	: Inactive by default
	Flexbox Container: Inactive by default
	Default to New Theme Builder: Active by default
	Landing pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default


== Log ==
	
PHP: showing 20 of 21PHP: 2022-04-13 16:25:41 [notice X 193][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/unlimited-elements-for-elementor-premium/provider/core/plugins/unlimited_elements/elementor/elementor_integrate.class.php(412): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-04-13 16:25:41 [notice X 193][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/unlimited-elements-for-elementor-premium/provider/core/plugins/unlimited_elements/elementor/elementor_integrate.class.php(416): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-04-13 16:25:41 [notice X 2942][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/e-addons-for-elementor/core/managers/controls.php(62): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-04-13 16:25:41 [notice X 491][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/e-addons-for-elementor/base/module-base.php(155): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-04-13 16:25:41 [notice X 193][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/query-control/module.php(826): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-04-13 16:25:41 [notice X 193][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/theme-builder/module.php(162): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-04-13 16:25:41 [notice X 193][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/forms/module.php(72): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-04-13 16:25:41 [notice X 193][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php(301): class type trigger_error()
#2: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(523): Elementor\Modules\DevTools\Deprecation -> deprecated_argument()
#3: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/managers/controls.php(501): Elementor\Controls_Manager -> register()
#4: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/forms/module.php(73): Elementor\Controls_Manager -> register_control()
',
)]
PHP: 2022-04-13 16:25:41 [warning X 390][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php::1277] Illegal string offset 'url' [array (
  'trace' => '
#0: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1277): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/unlimited-elements-for-elementor-premium/provider/core/plugins/unlimited_elements/helper_provider_core.class.php(613): Elementor\Controls_Stack -> parse_dynamic_settings()
#2: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/unlimited-elements-for-elementor-premium/provider/core/plugins/unlimited_elements/elementor/elementor_integrate.class.php(1487): HelperProviderCoreUC_EL :: putDynamicLoopElementStyle()
#3: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-includes/class-wp-hook.php(307): UniteCreatorElementorIntegrate -> onBeforeRenderElement()
#4: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-includes/class-wp-hook.php(331): WP_Hook -> apply_filters()
',
)]
PHP: 2022-04-27 10:39:46 [warning X 616][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php::1277] Illegal string offset 'url' [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-04-27 10:40:34 [error X 1][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/popup/document.php::61] Uncaught Error: Class 'ElementorPro\Modules\Popup\DisplaySettings\Triggers' not found in /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/popup/document.php:61
Stack trace:
#0 /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/popup/document.php(115): ElementorPro\Modules\Popup\Document->get_display_settings()
#1 /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/core/base/document.php(317): ElementorPro\Modules\Popup\Document->get_frontend_settings()
#2 /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(213): Elementor\Core\Base\Document->get_container_attributes()
#3 /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(435): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document->get_container_attributes()
#4 /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugin [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-04-27 10:40:35 [error X 2][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/post-excerpt.php::53] Uncaught Error: Call to undefined method ElementorPro\Core\Utils::trim_words() in /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/post-excerpt.php:53
Stack trace:
#0 /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/core/dynamic-tags/tag.php(35): ElementorPro\Modules\DynamicTags\Tags\Post_Excerpt->render()
#1 /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/core/dynamic-tags/manager.php(216): Elementor\Core\DynamicTags\Tag->get_content()
#2 [internal function]: Elementor\Core\DynamicTags\Manager->get_tag_data_content()
#3 /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/core/dynamic-tags/manager.php(109): call_user_func_array()
#4 /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/core/dynamic-tags/manager.php(74): Elementor\Core\DynamicTags\Manager->parse_tag_text()
#5 [internal function]: Elementor\Core\DynamicTags\Manager->Elementor\Core\DynamicTags\{closure}()
#6 / [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-04-27 10:49:54 [notice X 174][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/forms/classes/form-record.php::179] Undefined index: referrer [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-04-27 12:03:34 [notice X 11][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-text.php::33] Undefined offset: 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-04-28 05:47:25 [warning X 495][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php::1277] Illegal string offset 'url' [array (
  'trace' => '
#0: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php(1277): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/unlimited-elements-for-elementor-premium/provider/core/plugins/unlimited_elements/helper_provider_core.class.php(675): Elementor\Controls_Stack -> parse_dynamic_settings()
#2: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/unlimited-elements-for-elementor-premium/provider/core/plugins/unlimited_elements/elementor/elementor_integrate.class.php(1530): HelperProviderCoreUC_EL :: putDynamicLoopElementStyle()
#3: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-includes/class-wp-hook.php(307): UniteCreatorElementorIntegrate -> onBeforeRenderElement()
#4: /mnt/storage/home/exetergcc.co.uk/httpdocs/wp-includes/class-wp-hook.php(331): WP_Hook -> apply_filters()
',
)]
PHP: 2022-04-28 11:30:31 [warning X 37][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/base/controls-stack.php::1277] Cannot assign an empty string to a string offset [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-04-28 15:58:37 [notice X 9][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/includes/controls/base-multiple.php::87] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-05-04 17:52:00 [notice X 4][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-05-05 10:33:15 [notice X 922][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/modules/dev-tools/deprecation.php::301] The $control_id argument is deprecated since version 3.5.0! [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-05-09 12:30:11 [notice X 1][/mnt/storage/home/exetergcc.co.uk/httpdocs/wp-content/plugins/elementor/core/experiments/experiments-reporter.php::89] Undefined index: title [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 18 of 182022-04-27 10:38:02 [info] elementor::elementor_updater Started 
2022-04-27 10:38:02 [info] Elementor/Upgrades - _on_each_version Start  
2022-04-27 10:38:02 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.2',
  'to' => '3.6.4',
)]
2022-04-27 10:38:03 [info] Elementor/Upgrades - _on_each_version Finished 
2022-04-27 10:38:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.2',
  'to' => '3.6.4',
)]
2022-04-27 10:38:05 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.2',
  'to' => '3.6.4',
)]
2022-04-27 10:40:36 [info] elementor-pro::elementor_pro_updater Started 
2022-04-27 10:40:36 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-04-27 10:40:36 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-04-27 10:40:36 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Start  
2022-04-27 10:40:36 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.2',
  'to' => '3.6.5',
)]
2022-04-27 10:40:36 [info] Elementor Pro/Upgrades - _v_3_5_0_price_list Finished 
2022-04-27 10:40:36 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.2',
  'to' => '3.6.5',
)]
2022-04-27 10:40:37 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.4.2',
  'to' => '3.6.5',
)]
2022-05-05 10:32:34 [info] elementor::elementor_updater Started 
2022-05-05 10:32:34 [info] Elementor/Upgrades - _on_each_version Start  
2022-05-05 10:32:35 [info] Elementor/Upgrades - _on_each_version Finished 
2022-05-05 10:32:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.6.4',
  'to' => '3.6.5',
)]

JS: showing 4 of 4JS: 2022-05-02 12:30:43 [error X 1][https://exetergcc.co.uk/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5.e1:24:19952] View (cid: \"view102\") has already been destroyed and cannot be used. 
JS: 2022-05-04 09:54:08 [error X 7][https://exetergcc.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.4:2:515479] Cannot read properties of null (reading \'getBoundingClientRect\') 
JS: 2022-05-04 10:05:55 [error X 1][https://exetergcc.co.uk/wp-admin/post.php?post=20396&action=elementor#:1:1] Identifier \'$dcWidgetIdInput\' has already been declared 
JS: 2022-05-04 18:46:55 [error X 1][https://exetergcc.co.uk/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.4:2:381955] this.view.container is undefined 



== Elementor - Compatibility Tag ==
	
	e-addons for Elementor: Compatibility not specified
	Elementor Pro: Compatibility not specified
	Make Column Clickable Elementor: Compatibility not specified
	Unlimited Elements for Elementor (Premium): Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	

				

````

</details>

**Comment by eastriverstudio** [comment]
#18412

**Comment by eastriverstudio** [comment]
Image for reference showing correct responsive styles overidden by styles in inline file which is added also:

https://cln.sh/ZJKTSg

**Comment by eastriverstudio** [comment]
For anyone else who comes across this and are suffering similar issues I've had this back from Elementor's tier 2 support team and it is a confirmed bug with a workaround:

> Please note that the issue is caused by the "Additional Breakpoints" experiment when using a Single Post/Page template.
> Our developers are aware of the problem and are working on a fix which will be released as soon as possible. However, we don't have an ETA yet.
> 
> As a workaround, I've deactivated the Additional Breakpoint experiment. Would you please check if that resolved the issue?

Hope this helps :)

**Comment by hitsgee** [comment]
Same here

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core 3.15.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
